runtime/oti/j9javaaccessflags.h

```diff
@@ -30,112 +30,132 @@
 #define J9AccMethodHasMethodHandleInvokes 0x8000000
-#define J9AccMethodHasMethodParameters 0x800000
-#define J9AccMethodHasParameterAnnotations 0x40000000
 #define J9AccMethodHasStackMap 0x10000000
-#define J9AccMethodHasTypeAnnotations 0x4000000
-#define J9AccMethodObjectConstructor 0x400000
+#define J9AccMethodHasMethodAnnotations 0x20000000
+#define J9AccMethodHasParameterAnnotations 0x40000000
+#define J9AccMethodHasDefaultAnnotation 0x80000000
+/* Masks and helpers */
+#define J9AccMethodReturnMask 0x1C0000
+#define J9AccMethodReturnShift 0x12
+/* Used in HookedByTheJit.cpp */
 #define J9AccMethodReturn0 0x0
 #define J9AccMethodReturn1 0x40000
 #define J9AccMethodReturn2 0x80000
 #define J9AccMethodReturnA 0x140000
 #define J9AccMethodReturnD 0x100000
 #define J9AccMethodReturnF 0xC0000
-#define J9AccMethodReturnMask 0x1C0000
-#define J9AccMethodReturnShift 0x12
-#define J9AccMethodVTable 0x10000
-#define J9AccNative 0x100
-#define J9AccPrivate 0x2
-#define J9AccProtected 0x4
-#define J9AccPublic 0x1
-#define J9AccStatic 0x8
-#define J9AccStrict 0x800
-#define J9AccSuper 0x20
-#define J9AccSynchronized 0x20
-#define J9AccSynthetic 0x1000
-#define J9AccTransient 0x80
-#define J9AccVarArgs 0x80
+/* DDR flags */
+#define J9AccMethodHasTypeAnnotations 0x4000000
+
+/* Field flags */
 #define J9AccVolatile 0x40
-#define J9AccValueType 0x40
-#define J9AccRecord 0x400
-#define J9AccSealed 0x200
-#define J9AccPrimitiveValueType 0x800
+#define J9AccTransient 0x80
+
+/* RAM class classDepthAndFlags.
+ * See map in createramclass.cpp:internalCreateRAMClassFromROMClassImpl
+ * for available slots.
+ */
+#define J9AccClassRAMArray 0x10000
+#define J9AccClassHasBeenOverridden 0x100000
+#define J9AccClassOwnableSynchronizer 0x200000
+#define J9AccClassHasJDBCNatives 0x400000
+#define J9AccClassGCSpecial 0x800000
+#define J9AccClassContinuation 0x1000000
+#define J9AccClassHotSwappedOut 0x4000000
+#define J9AccClassDying 0x8000000
+/* Masks and helpers */
+#define J9AccClassDepthMask 0xFFFF
+#define J9AccClassReferenceMask 0x30000000
+#define J9AccClassRomToRamMask 0xF3000000
+#define J9AccClassRAMShapeShift 0x10
+#define J9_JAVA_CLASS_FINALIZER_CHECK_MASK (J9AccClassFinalizeNeeded | J9AccClassOwnableSynchronizer | J9AccClassContinuation)
+#define J9_JAVA_MODIFIERS_SPECIAL_OBJECT (J9AccClassFinalizeNeeded | J9AccClassReferenceMask)
+
+/* static field helpers*/
 #define J9StaticFieldRefBaseType 0x1
 #define J9StaticFieldRefDouble 0x2
 #define J9StaticFieldRefVolatile 0x4
 #define J9StaticFieldRefBoolean 0x8
 #define J9StaticFieldRefPutResolved 0x10
-#define J9StaticFieldRefFinal 0x20
 #define J9StaticFieldRefFlagBits 0x3F
-#define J9_JAVA_CLASS_FINALIZER_CHECK_MASK (J9AccClassFinalizeNeeded | J9AccClassOwnableSynchronizer | J9AccClassContinuation)
-#define J9_JAVA_MODIFIERS_SPECIAL_OBJECT (J9AccClassFinalizeNeeded | J9AccClassReferenceMask)
+#define J9StaticFieldRefFinal 0x20
 
-#endif /*J9JAVAACCESSFLAGS_H */
+#endif /*J9JAVAACCESSFLAGS_H */
```

Comment:
Please restore the newline at the end of file.

Reply:
Fixed.